app/models/community_translation.rb

```diff
@@ -0,0 +1,25 @@
+# == Schema Information
+#
+# Table name: community_translations
+#
+#  id              :integer          not null, primary key
+#  community_id    :integer          not null
+#  locale          :string(16)       not null
+#  translation_key :string(255)      not null
+#  translation     :string(255)
```

Comment:
Since we are not using this as a search key, I guess we could go just with text?